package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "build": "webpack",
+    "build": "NODE_ENV=production webpack",
```

Comment:
Is there any need to build in non-prod environments? If so should we create another script called `build:dev` or something?

Reply:
Good catch, our npm run dev command will handle that 
```
   "dev": "NODE_ENV=development concurrently \"npm run serve:watch\" \"npm run build:watch\"",
```